src/debugger/debugger.ts

```diff
@@ -59,19 +59,43 @@ async function getBinaryPath(): Promise<string | undefined> {
 	}
 }
 
-export async function connectToDebugger(functionName: string) {
+export async function connectToDebugger(
+	functionName: string,
+	moduleName: string,
+	fileName: string,
+) {
 	// The binary that is being referred to here, is the binary present in the cargo artifacts.
 	// It looks like this - kani_concrete_playback_check_estimate_size_14615086421508420155
 	const binaryName = await getBinaryPath();
 
+	let functionParameter: string;
+
+	if (fileName === 'main') {
+		if (moduleName !== '') {
+			functionParameter = `${moduleName}::${functionName}`;
+		} else {
+			functionParameter = functionName;
+		}
+	} else if (fileName != '') {
+		if (moduleName === '') {
+			functionParameter = `${fileName}::${functionName}`;
```

Comment:
File name may not imply the module. For example, `mod.rs` usually implies that the module is the name of the folder where it is contained.

Another example is lib.rs.